qemu/tests/cfg/nvdimm.cfg

```diff
@@ -132,6 +132,7 @@
                     type = boot
                     unarmed_dimm_mem1 = on
         - nvdimm_nvml:
+            no ppc64, ppc64le
```

Comment:
Do we need to restrict the version to disable?

Reply:
Per David, we don't need to, thanks.